### PR TITLE
data privacy option for OMP usage stats plugin

### PR DESCRIPTION
**Pull request by bozana**
I would like to make the pull request for this branch, if someone would like to use it immediatelly...
I will then port it to master too, although the master branch seems to currently be in havy development...

### CODE REVIEW

**Comment by asmecher** [comment]
@beghelli, suddenly a flurry of activity on stuff you know best... Mind taking a look? Thanks, @bozana!

**Comment by bozana** [comment]
could you cherry-pick the first commit (the second is already there) to the master or shall I make a new pull request?

File: plugins/generic/usageStats/README [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Data Privacy Option
+--------------------
+
+REQUIREMENTS:
+In order to use the data privacy option, insert the section "[usageSats]" in the
```
Comment (beghelli) [inline plugins/generic/usageStats/README]: typo, usageSats instead of usageStats

File: plugins/generic/usageStats/PKPUsageStatsPlugin.inc.php [hunk 2]
```diff
@@ -186,6 +185,10 @@ function callbackLoadCategory($hookName, $args) {
 			$this->import('UsageStatsReportPlugin');
 			$plugin = new UsageStatsReportPlugin();
 		}
+		if ($category ==  'blocks') {
+			$this->import('UsageStatsOptoutBlockPlugin');
+			$plugin = new UsageStatsOptoutBlockPlugin($this->getName());
```
Comment (beghelli) [inline plugins/generic/usageStats/PKPUsageStatsPlugin.inc.php]: I didn't see that before, but this block will always appear, no matter if the manager has activated the data privacy option or not right? Is this what you wanted?

Comment (bozana) [inline plugins/generic/usageStats/PKPUsageStatsPlugin.inc.php]: Ah, I could check here if the option is chosen, right? Thanks Bruno!!! I will then change those two things (the typo too) and let you know...

**Comment by asmecher** [comment]
@beghelli and @bozana, I'm guessing this still needs to be forward-ported to the master branch? If so, and if you're not working on that at the moment, could one of you file an issue so it doesn't get forgotten?

**Comment by beghelli** [comment]
we can track that on https://github.com/pkp/pkp-lib/issues/685 I think...

**Pull request merged**